src/libs/ck-libs/collide/bbox.h

```diff
@@ -14,181 +14,181 @@ Rather complete interval and box classes.
+  void setMax(T m) {max=m;}
+
+  //Set this span to contain no points
+  seg1d &empty(void) {
+    min=2000000000; max=-2000000000; return *this;
+  }
+  //Set this span to contain all points
+  seg1d &infinity(void) {
+    min=-2000000000; max=2000000000; return *this;
+  }
+
+  bool isEmpty(void) const { return max<min;}
+  //Set this span to contain only this point
+  seg1d &set(T b) {min=max=b; return *this;}
+  //Set this span to contain these two points and everything in between
+  seg1d &set(T a,T b) {
+    if (a<b) {min=a;max=b;}
+    else     {min=b;max=a;}
+    return *this;
+  }
+  //Expand this span to contain this point
+  void expandMin(T b) {if (min>b) min=b;}
+  void expandMax(T b) {if (max<b) max=b;}
+  seg1d &add(T b) {
+    expandMin(b);expandMax(b);
+    return *this;
+  }
+  //Expand this span to contain this span
+  seg1d &add(const seg1d &b) {
+    expandMin(b.min);expandMax(b.max);
+    return *this;
+  }
+
+  //Return the intersection of this and that seg
+  seg1d getIntersection(const seg1d &b) const {
+    return seg1d(min>b.min?min:b.min, max<b.max?max:b.max);
+  }
+  //Return the union of this and that seg
+  seg1d getUnion(const seg1d &b) const {
+    return seg1d(min<b.min?min:b.min, max>b.max?max:b.max);
+  }
+
+  //Return true if this seg contains this point
+  // in its interior or boundary (closed interval)
+  bool contains(T b) const {
+    return (min<=b)&&(b<=max);
+  }
+  //Return true if this seg contains this point
+  // in its interior (open interval)
+  bool containsOpen(T b) const {
+    return (min<b)&&(b<max);
+  }
+  //Return true if this seg contains this point
+  // in its interior or left endpoint (half-open interval)
+  bool containsHalf(T b) const {
+    return (min<=b)&&(b<max);
+  }
+  //Return true if this seg and that share any points
+  bool intersects(const seg1d &b) const {
+    return contains(b.min)||b.contains(min);
+  }
+  //Return true if this seg and that share any interior points
+  bool intersectsOpen(const seg1d &b) const {
+    return containsHalf(b.min)||b.containsOpen(min);
+  }
+  //Return true if this seg and that share any half-open points
+  bool intersectsHalf(const seg1d &b) const {
+    return containsHalf(b.min)||b.containsHalf(min);
+  }
+
+  inline void pup(PUP::er &p) {
+    p|min;
+    p|max;
+  }
 };
 typedef seg1dT<double> rSeg1d;
 typedef seg1dT<int> iSeg1d;
 
 class bbox3d {
-	rSeg1d segs[3];//Spans for x (0), y (1), and z (2)
-public:   
-	bbox3d() {}
-	bbox3d(const rSeg1d &x,const rSeg1d &y,const rSeg1d &z)
-		{segs[0]=x; segs[1]=y; segs[2]=z;}
-	bbox3d(const CkVector3d &a,const CkVector3d &b,const CkVector3d &c)
-	{
-		segs[0].init(a[0],b[0],c[0]);
-		segs[1].init(a[1],b[1],c[1]);
-		segs[2].init(a[2],b[2],c[2]);
-	}
-	
-	void print(const char *desc=NULL) const;
+  rSeg1d segs[3];//Spans for x (0), y (1), and z (2)
+  public:
+  bbox3d() {}
+  bbox3d(const rSeg1d &x,const rSeg1d &y,const rSeg1d &z)
+  {segs[0]=x; segs[1]=y; segs[2]=z;}
```

Comment:
Did something go wrong with the whitespace cleanup here?

Reply:
It looks like `gg=G` indents `{` right under the function definition start and that's what was done here. It just happened to have a one-line body causing the weirdness. I fixed this in the next commit.